elasticdl/python/ps/parameter_server.py

```diff
@@ -28,6 +29,10 @@ def __init__(self, args):
         self.optimizer = model_module[args.optimizer]()
         # Create Parameters instance
         self.parameters = Parameters()
+        if args.master_addr is None:
+            raise ValueError("master_addr is missing for worker")
```

Comment:
master_addr is missing for parameter servers.

Reply:
Done